src/RedirectMiddleware.php

```diff
@@ -25,11 +25,12 @@ class RedirectMiddleware
      * @var array
      */
     public static $defaultSettings = [
-        'max'             => 5,
-        'protocols'       => ['http', 'https'],
-        'strict'          => false,
-        'referer'         => false,
-        'track_redirects' => false,
+        'max'                      => 5,
+        'protocols'                => ['http', 'https'],
+        'strict'                   => false,
+        'referer'                  => false,
+        'track_redirects'          => false,
+        'force_pass_authorization' => false,
```

Comment:
I don't like this name. How about `forward_authorization` or even `forward_authorization_on_redirect`. I suppose there's no real reason to have a short string over clarity?

Reply:
👍  for `forward_authorization_on_redirect`

Reply:
I also think this should have 3 possible values. `false`, `null` or `true`. Where `null` passes where the Host is the same, and `false` never passes, and `true` always passes.